pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -29,7 +29,14 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-var log = logs.GetLogger("kieapp.defaults")
+var (
+	log          = logs.GetLogger("kieapp.defaults")
+	caOptsAppend = []string{
```

Comment:
in case the user needs to provide a new TrustStore to access internal services outside ocp, is there a way to append the user's provided TrustStore to this one which will be used to configure the JVM?

Reply:
i don't believe so... would be an either or kind of thing

Reply:
that said, if they added their trusted CA certs to the openshift cluster

Reply:
ok, we might need to add a note about it on the docs just to make users aware and update it on future releases, wdyt?

Reply:
sure... i mean, maybe we could add a way to add a user-provided truststore as well in the future? but it felt like this was a big step forward using openshift's ca bundle.

Reply:
yep, let's track it on a future improvement.